config/charts/body-based-routing/templates/istio.yaml

```diff
@@ -25,9 +25,9 @@ spec:
           processing_mode:
             request_header_mode: "SEND"
             response_header_mode: "SKIP"
-            request_body_mode: "BUFFERED"
+            request_body_mode: "FULL_DUPLEX_STREAMED"
             response_body_mode: "NONE"
```

Comment:
Your changes slightly differ from https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/552. For example, the response body should be `FULL_DUPLEX_STREAMED`.

Reply:
BBR doesn't need the response body like EPP so it should be okay to omit.